tests/sample/test_sample_models.py

```diff
@@ -0,0 +1,149 @@
+from datetime import datetime
+import uuid
+
+from dataservice.extensions import db
+from dataservice.api.participant.models import Participant
+from dataservice.api.sample.models import Sample
+from tests.utils import FlaskTestCase
+
+from sqlalchemy.exc import IntegrityError
+
+class ModelTest(FlaskTestCase):
```

Comment:
If the relationship between Participant and Sample is actually one to many then these tests will need to account for that. Should test adding multiple samples to a Participant, updating a single sample in the list of Participant's samples, deleting one sample from the list of Participant's samples, etc